lib/rectangles.js

```diff
@@ -0,0 +1,24 @@
+function area(rectangle) {
+  return rectangle.height * rectangle.width
+}
+
+function perimeter(rectangle) {
+  return 2 * (rectangle.height + rectangle.width)
+}
+
+function diagonal(rectangle) {
+  var heightSquared = rectangle.height ** 2
+  var widthSquared = rectangle.width ** 2
+  return Math.sqrt(heightSquared + widthSquared)
+}
+
+function isSquare(rectangle) {
+  return rectangle.height === rectangle.width ? true : false
```

Comment:
love the one liner here! You can make it similar by just `return rectangle.height === rectangle.width` since that already returns a boolean for us